README.md

```diff
@@ -1,107 +1,131 @@
 
-One big issue we noted during our Gazelle-plugin development is that we can't easily and exactly reproduce a Spark stage. Once we meet some bugs during Spark run, Gazelle-plugin doesn't dump enough info to reproduce it natively. Mainly because we use very complex extended Gandiva tree to pass the query plan. With well defined substrait and some helper functions, we can easily reproduce the whole stage, which makes debug, profile and optimize the native code much more easier. It also make the accelerators enabling much more easier even without touching Spark code.
+## 1.2 Gluten's Solution
 
-![Overview](./docs/image/reproduce_natively.png)
+“Gluten” is Latin for glue. Main goal of project Gluten is to “glue" the SparkSQL and native libraries. So we can take use of and benefit from Spark SQL's scale out framework as well native libraries' high performance.
 
-# Plan Build
+The basic rule of Gluten's design is that we would reuse spark's whole control flow and as many JVM code as possible but offload the compute intensive data processing part to native code. Here is what Gluten does:
+* Transform Spark’s whole stage physical plan to Substrait plan and send to native
+* Offload performance critical data processing to native library
+* Define clear JNI interfaces for native libraries
+* Switch the native backends easily
+* Reuse Spark’s distributed control flow
+* Manage data sharing between JVM and native
+* Extend support to native accelerators
 
-To convert Spark's physical plan into Substrait plan, we defined a substrait transformer operator which is a wrapper of the tree of operators to be executed natively. The operator's doTransform function return the final substrait tree. doExecutorColumnar function execute the native node and return columnarBatch. Each operator has its own transformerExec which transforms this operator's plan node into a substrait plan node by transform function. The validate function is designed to check if native library support the operator. The whole process is very like Spark's whole stage code generation flow.
+## 1.3 Target User
 
-![Overview](./docs/image/operators.png)
+Gluten targets to the Spark administrators and Spark users who want to improve their spark cluster's performance fundamentally. Gluten is an plunin to Spark. It's designed to offload the SQL engine to native without any dataframe API or SQL query changes. SparkSQL users can run their current Spark job on Gluten seemlessly, no code changes are needed. However as an plugin, Gluten needs some configurations to enable it when you start Spark context. All configurations are listed [here](https://github.com/oap-project/gluten/blob/main/docs/Configuration.md)
```

Comment:
nit: (1) Gluten is `an plunin` to Spark -> a plugin  (2) However as `an` plugin -> a plugin.